src/sql-parser/src/parser.rs

```diff
@@ -7199,6 +7202,45 @@ impl<'a> Parser<'a> {
             query,
         }))
     }
+    /// Parse an `EXPLAIN [KEY|VALUE] SCHEMA` statement assuming that the `EXPLAIN [KEY|VALUE]` tokens
+    /// have already been consumed
+    fn parse_explain_schema(&mut self) -> Result<Statement<Raw>, ParserError> {
```

Comment:
We have a bunch of parser specific tests that we like to add for new statement types. Would you mind adding some test for this? You can find some examples here: https://github.com/MaterializeInc/materialize/blob/main/src/sql-parser/tests/testdata/explain, feel free to reach out if you have any questions.